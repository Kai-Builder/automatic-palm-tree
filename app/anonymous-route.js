import Anonymous from './anonymous';
import React from 'react';
// import {SecureRoute} from '@okta/okta-react';
import {Route as SecureRoute} from 'react-router-dom';

const AnonymousRoute = () => (
    <SecureRoute
        component={Anonymous}
        path='/anonymous'
    />
);

export default AnonymousRoute;
