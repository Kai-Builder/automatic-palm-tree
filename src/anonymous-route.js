import Anonymous from './anonymous';
import React from 'react';

// import {SecureRoute} from '@okta/okta-react';

import {
  Route
} from 'react-router-dom';

const AnonymousRoute = () => (
  <Route
    component={Anonymous}
    path='/anonymous'
  />
);

export default AnonymousRoute;
