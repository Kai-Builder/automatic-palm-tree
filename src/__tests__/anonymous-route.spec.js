import {AnonymousRoute} from '../anonymous-route';
import React from 'react';
import {shallow} from 'enzyme';

describe('AnonymousRoute', () => {
  const requiredProps = () => ({
  });
  const render = (props = requiredProps()) => shallow(<AnonymousRoute {...props}/>);

  test('component renders', () => {
    const wrapper = render();

    expect(wrapper).toMatchSnapshot();
  });
});
