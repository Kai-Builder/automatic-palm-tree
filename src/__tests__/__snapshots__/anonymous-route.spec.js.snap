// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnonymousRoute component renders 1`] = `
<Route
  component={[Function]}
  path="/anonymous"
/>
`;

exports[`AnonymousRoute component to be open for non authorized users 1`] = `
<Route
  component={[Function]}
  path="/anonymous"
/>
`;

exports[`AnonymousRoute path is correct 1`] = `
<Route
  component={[Function]}
  path="/anonymous"
/>
`;

exports[`AnonymousRoute route renders Anonymous 1`] = `
<Route
  component={[Function]}
  path="/anonymous"
/>
`;
